kubevirt-cdi/step1.md

```diff
@@ -1,58 +1,55 @@
 
-In this scenario, we'll go quickly over installing KubeVirt as it was covered already in the introduction scenario: 'First steps with KubeVirt'
+In this exercise we start by deploying the CDI operator. Then, we import a CirrOS disk image and use it to start a VM.
 
-We query GitHub's API to get the latest available release (click on the text to auto-execute the commands on the console):
+# Wait for KubeVirt to deploy
 
-`export KUBEVIRT_VERSION=$(curl -s https://api.github.com/repos/kubevirt/kubevirt/releases/latest | jq -r .tag_name)
-echo $KUBEVIRT_VERSION`{{execute}}
+The setup for this scenario includes installation of KubeVirt and the `virtctl` utility.
 
-Run the following command to deploy the KubeVirt Operator:
+Before we can start, we need to wait for the KubeVirt initialization script to run. (a command prompt will appear once everything is ready).
 
-`kubectl create -f https://github.com/kubevirt/kubevirt/releases/download/${KUBEVIRT_VERSION}/kubevirt-operator.yaml`{{execute}}
+# Install Hostpath Provisioner
 
-Now let's deploy KubeVirt by creating a Custom Resource that will trigger the 'operator' reaction and perform the deployment:
+Before we can install CDI, we have some prerequisites, namely a supported storage class and provisioner. For this example, we use the Hostpath provisioner which provisions PVCs using node local storage. This is an option for proof of concept exercises like this one, but should not be used in production because it does not support RWX nor accessing a volume across nodes.
 
-`kubectl create -f https://github.com/kubevirt/kubevirt/releases/download/${KUBEVIRT_VERSION}/kubevirt-cr.yaml`{{execute}}
+The setup for this scenario includes installation of the Hostpath Provisioner.
 
-Next, we need to configure KubeVirt to use software emulation for virtualization. This is necessary for the Katacoda environment, but results in poor performance, so avoid this step in production environments.
+`kubectl get storageclass`{{execute}}
 
-`kubectl -n kubevirt patch kubevirt kubevirt --type=merge --patch '{"spec":{"configuration":{"developerConfiguration":{"useEmulation":true}}}}'`{{execute}}
+Before we continue, we need to make sure the Hostpath Provisioner has completely deployed:
 
-Let's check the deployment:
+# Install the Containerized Data Importer
 
-`kubectl get kubevirt -n kubevirt`{{execute}}
+Next we determine the latest version of CDI and apply both the Operator and the CR that starts the deployment:
 
-It will take a while until all the pods are running. Retry the command until the output states that kubevirt is "Deployed".
+```
+export VERSION=$(curl -Ls https://github.com/kubevirt/containerized-data-importer/releases/latest | grep -m 1 -o "v[0-9]\.[0-9]*\.[0-9]*")
+echo $VERSION
+```{{execute}}
```

Comment:
the {{execute}} may be messing with the three back tics. The remaining page does not render correctly.